dmoj/judge.py

```diff
@@ -77,6 +77,12 @@ def __init__(self, packet_manager: packet.PacketManager) -> None:
         self.updater_signal = threading.Event()
         self.updater = threading.Thread(target=self._updater_thread)
 
+        self.in_pretests = False
+        self.pretest_batch = 0
```

Comment:
Where do we reset these values?

Reply:
Moved these values to be initialized in the `grading_begin` packet.